vendor/models/offer.py

```diff
@@ -206,6 +206,9 @@ def get_next_billing_date(self):
             start_date = self.term_start_date
         return get_payment_scheduled_end_date(self, start_date)
 
+    def get_subscription_start_date(self, profile):
```

Comment:
I don't think it is a good idea to just create a function to call a function. why just not call it where you are using it?